### PR TITLE
Fix request method bug

### PR DESCRIPTION
**Pull request by kc-si**

### CODE REVIEW

**Review by marekciupak — APPROVED** [review]
Nice :clap: Looks good to me.

File: config/application.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module Jsclook
+module JSClook
```
[the remark this answers is not in the document]
Comment (kc-si) [reply config/application.rb:23]: Can't ignore this, I thought I did it :) will change it

File: app/views/layouts/_session.html.erb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <% if user_signed_in? %>
-  <div class='text-secondary d-inline'><%= current_user.email %></div>
-  <%= link_to 'Logout', destroy_user_session_path, class: "btn btn-outline-secondary" %>
+  <div class="d-flex flex-nowrap">
+    <%= link_to current_user.email, '#', class: "nav-link" %>
+    <%= button_to 'Logout', destroy_user_session_path, method: :delete, class: "btn btn-outline-secondary d-inline" %>
+  </div>
 <% else %>
-  <%= link_to 'Login', new_user_session_path, class: "btn btn-outline-success" %>
+  <%= button_to 'Login', new_user_session_path, class: "btn btn-outline-success" %>
```
[the remark this answers is not in the document]
Comment (kc-si) [reply app/views/layouts/_session.html.erb:7]: I was thinking about, but I chose wrong,
lesson for next time

File: config/routes.rb [hunk 3]
```diff
@@ -5,8 +5,9 @@
   # Defines the root path route ("/")
   root 'companies#index'
 
-  get '/companies/update_companies_list', to: 'companies#update_companies_list'
-  get '/companies/update_companies_details', to: 'companies#update_companies_details'
+  patch '/companies/update_companies_list', to: 'companies#update_companies_list'
+  patch '/companies/update_companies_details', to: 'companies#update_companies_details'
```
[the remark this answers is not in the document]
Comment (kc-si) [reply config/routes.rb:9]: Yes, that is correct, changing it

**Pull request merged**